### DISCUSSION TITLE
Option to save / switch between 2 or more offsets

### DISCUSSION THREAD

**Discussion by Bmjoe** [ppy/osu#30768] [Ideas]
Hello,

the idea is being able to save an offset, such as -42 ms, and load or switch to it at a later time, just like one would save and load a skin. 

Because I play without hitsounds, I try to adjust the offset so that it syncs with the music; the problem is that every device has a different input lag, so if I switch between my headphone amplifier and my speaker amplifier, the offset isn't correct anymore and I'll have to try and play for a good 1-2 hours to get it right. 

This could be solved by labeling and saving an offset, such as "headphone offset 1" or "speaker offset 2" and so on. 

Some players may also want to switch between offsets depending on their mood, or just try to switch back and forth between several offsets quickly to experiment etc. and this option could help immensely with that. 

Thanks!

**Comment by peppy** [comment]
We may introduce settings profiles which can be saved. This would be at a game level, including all settings rather than drilling down to something as specific as audio offset, but should cover your use case.

**Reply by N0Y0U2114** [reply]
Something that A dance of fire and ice has is device specific audio offset. Whenever you use an audio device, it saved the audio offset for the device, and whenever the same device is used, it swaps back to that profile.
